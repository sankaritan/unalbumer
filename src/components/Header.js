import React from "react";

const Header = () => (
    <div>
        <h4 id='app-title'>Unalbumer</h4>
        <hr></hr>
    </div>
)

export default Header;