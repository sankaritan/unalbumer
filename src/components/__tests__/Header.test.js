import React from "react";
import Header from "../Header";
import Adapter from "enzyme-adapter-react-16";
import Enzyme, { shallow } from "enzyme";

Enzyme.configure({ adapter: new Adapter() });

describe("Header component", () => {
  it("component renders correctly", () => {
    const tree = shallow(<Header />).debug();
    expect(tree).toMatchSnapshot();
  });
});
