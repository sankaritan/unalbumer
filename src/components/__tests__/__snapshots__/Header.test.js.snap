// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component component renders correctly 1`] = `
"<div>
  <h4 id=\\"app-title\\">
    Unalbumer
  </h4>
  <hr />
</div>"
`;
